index.js

```diff
@@ -235,12 +236,12 @@ function calculateRange(text, opts, ast) {
   const startNodeAndParents = findNodeAtOffset(
     ast,
     startNonWhitespace,
-    isSourceElement
+    isSourceElement.bind(null, opts)
```

Comment:
Sure, fixed in https://github.com/prettier/prettier/pull/2298/commits/e06dc7943ec7c63c71880d45a39475192ec57bc8